protobuf/avs.proto

```diff
@@ -576,6 +576,7 @@ message TriggerReason {
     Cron      = 4;
     Block     = 5;
     Event     = 6;
+    Default   = 7;  // Used as default when no specific trigger type is provided
```

Comment:
Remove this addition and reuse the value `0` as the default value. What would be a better name than `Unset`?